src/components/EventCalender/Monthly/EventCalender.tsx

```
@@ -330,18 +332,22 @@ const Calendar: React.FC<
             </ul>
           </div>
 
-          <div className={styles.events_card} role="region" aria-label="Events">
-            <h3 className={styles.card_title}>Events</h3>
+          <div
+            className={styles.events_card}
+            role="region"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<section>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)